components/hab/habitat/plan.ps1

```diff
@@ -5,8 +5,18 @@ $pkg_maintainer = "The Habitat Maintainers <humans@habitat.sh>"
 $pkg_license = @("Apache-2.0")
 $pkg_source = "https://s3-us-west-2.amazonaws.com/habitat-win-deps/hab-win-deps.zip"
 $pkg_shasum="00b34fb983ebc43bfff9e8e2220d23db200cb45494a4971a5e2e733f1d73d04b"
+$pkg_deps=@(
+    "core/openssl",
+    "core/libarchive",
+    "core/libsodium"
+)
 $pkg_bin_dirs = @("bin")
-$pkg_build_deps = @("core/visual-cpp-redist-2013", "core/rust", "core/cacerts")
+$pkg_build_deps = @(
+    "core/visual-cpp-redist-2013",
```

Comment:
I **think** this 2013 dep can go since everything is compiled via the 2015 runtime.